asab/web/authz/service.py

```diff
@@ -24,29 +35,40 @@ class AuthzService(asab.Service):
 
 	def __init__(self, app, service_name="asab.AuthzService"):
 		super().__init__(app, service_name)
+		self.RBACDisabled = asab.Config.getboolean("authz", "_disable_rbac")
+		self._TokenVerificationDisabled = asab.Config.getboolean("authz", "_disable_token_verification")
+		if jwcrypto is None and not self._TokenVerificationDisabled:
+			raise ModuleNotFoundError(
+				"You are trying to use asab.web.authz without 'jwcrypto' installed. "
+				"Please run 'pip install jwcrypto' "
+				"or install asab with 'authz' optional dependency.")
+		self.PublicKeysUrl = asab.Config.get("authz", "public_keys_url")
+		if len(self.PublicKeysUrl) == 0 and not self._TokenVerificationDisabled:
+			raise ValueError("No public_keys_url provided in [authz] config section.")
+		self.AuthServerPublicKey = None  # TODO: Support multiple public keys
+		self.App.PubSub.subscribe("Application.tick/30!", self._fetch_public_keys_if_needed)
```

Comment:
The more sustainable strategy is to fetch keys when validation fails.
That is easy extension to this code - set self.AuthServerPublicKey to be None

Reply:
ok, i added a todo